cloud/shared/bin/lib/config_loader.py

```diff
@@ -70,6 +71,27 @@ def _load_config_fields(self, config_file: str):
         """
         config_parser = ConfigParser()
         config_fields = config_parser.parse_config(config_file)
+
+        if config_fields.get(
+                "ALLOW_ADMIN_WRITEABLE") and not os.getenv('SKIP_USER_INPUT'):
+            msg = inspect.cleandoc(
+                """
+                ###########################################################################
```

Comment:
**nitpick:**  Thanks for adding this! You could also consider using the colors like [here](https://github.com/civiform/cloud-deploy-infra/blob/c6ad29d40036edca377d7d9caf84c602a2ccd8d1/cloud/aws/bin/deploy.py#L68).

Reply:
Ooooo I love that idea.